esmf-semantic-aspect-meta-model/src/main/resources/samm/characteristic/2.0.0/characteristic-shapes.ttl

```diff
@@ -783,3 +783,27 @@ samm-c:FixedPointConstraintShape
       sh:name "integer" ;
       sh:description "The number of integral digits for a fixed point number." ;
    ] .
+
+samm-c:TimeSeriesShape
+   a sh:NodeShape ;
+   sh:targetClass samm-c:TimeSeries ;
+   sh:property [
+      sh:path samm:dataType ;
+      sh:minCount 0 ;
+      sh:maxCount 1 ;
+      sh:name "dataType" ;
+      sh:description "This constraint for TimeSeries type and for checking extending this type from TimeSeriesEntity." ;
+      sh:sparql [
+         a sh:SPARQLConstraint ;
+         sh:message "TimeSeries Entry to be extended from TimeSeriesEntity. You can use this structure for resolve it: 'samm:extends samm-e:TimeSeriesEntity ;'" ;
```

Comment:
This text also needs some improvement, it also should name the concrete element (via {} - see the other error messages in the file) which is invalid to allow the users to quickly find it.